src/Symfony/Component/VarExporter/LazyGhostTrait.php

```diff
@@ -379,13 +379,4 @@ public function __destruct()
             parent::__destruct();
         }
     }
-
-    private function setLazyObjectAsInitialized(bool $initialized): void
```

Comment:
wrong removal, a private method on a trait can still be used by consumers

Reply:
I didn't know ! Thanks, it's reverted